src/sentry/static/sentry/app/views/settings/projectAlerts/ruleEditor/ruleNodeList.jsx

```diff
@@ -35,19 +40,15 @@ class RuleNodeList extends React.Component {
           })}
         </RuleNodes>
         <fieldset>
-          <SelectInput
-            onChange={sel2 => this.props.handleAddRow(sel2.val())}
+          <StyledSelect
```

Comment:
does this one need to be searchable?

Reply:
it seems to work, although it is weird to have multiple types of dropdown on this page

Reply:
I'll revert this one for now